src/frontend/packages/core/src/shared/components/app-action-monitor/app-action-monitor.component.ts

```diff
@@ -82,9 +81,16 @@ export class AppActionMonitorComponent<T> implements OnInit {
       cellFlex: '0 0 24px'
     };
 
+    // Some obs will only ever emit once, once consumed in template this meant table never received emitted data
+    // so wrap in publish replay
+    const replayData = this.data$.pipe(
+      publishReplay(1),
+      refCount()
+    )
+
     this.allColumns = [...this.columns, monitorColumn];
     this.dataSource = {
-      connect: () => this.data$,
+      connect: () => replayData,
```

Comment:
Performance? Use another observable instead of `replayData` and return this here